Libplanet/Net/DifferentProtocolVersionEventArgs.cs

```diff
@@ -2,7 +2,7 @@ namespace Libplanet.Net
 {
     /// <summary>
     /// The event data that provides a value when
-    /// <see cref="Swarm{T}.DifferentVersionPeerEncountered" /> is occurred.
+    /// <see cref="NetMQTransport.DifferentVersionPeerEncountered" /> is occurred.
```

Comment:
I guess `DiffrentVersionPeerEncountered` isn't related to specific transport like `NetMQTransport`.

Reply:
You are right. I left `FIXME` in protocol detection part in `NetMQTransport` that it should be in `Swarm<T>`.